test/test_cursor.py

```diff
@@ -22,6 +22,8 @@
 import time
 import threading
 
+from typing import no_type_check
```

Comment:
Is this needed? test_cursor seems to type check without errors locally for me.

Reply:
Yeah it works fine with the relaxed rules

Reply:
(I removed it)